packages/dds/tree/src/test/simple-tree/arrayNode.spec.ts

```diff
@@ -750,35 +750,6 @@ describe("ArrayNode", () => {
 	describeHydration(
 		"Iteration",
 		(init) => {
-			it("Iterator of an unhydrated node works after it's been inserted, and throws during iteration once a concurrent edit is made.", () => {
```

Comment:
this test did not use the init function provided by describeHydration so it has been moved to avoid running it twice.